zebra-network/src/meta_addr.rs

```diff
@@ -831,7 +830,7 @@ impl Ord for MetaAddr {
         // So this comparison will have no impact until Zebra implements
         // more service features.
```

Comment:
Documentation: Can we add a note that the `Option<PeerServices>` ordering still works correctly here?

I've also added a note to #2324 to re-check this ordering.

```suggestion
        // So this comparison will have no impact until Zebra implements
        // more service features.
        //
        // None is less than Some(T), so peers with missing services are chosen last.
```